python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -171,6 +174,8 @@ export default function EditorContainer({
   const stateRef = useRef(aiconfigState);
   stateRef.current = aiconfigState;
 
+  const logEvent = callbacks.logEvent;
```

Comment:
nit: a bit confusing since this isn't the `logEvent`, it's a `logEventCallback`, can we rename it to that (as well as prop name within `AIConfigContext`)?

Reply:
updated per your suggestion